ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/ExceptionRequestTableCells.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
```

Comment:
I was debating whether to separate each component into a different file but left it for now. If you think readability would be better as separate files, I can change that.

Reply:
Actually, for the sake of testing, maybe it would be better to separate them so I can make separate test files for each

Reply:
I think this looks good as a single file, but separate would be OK too. The only components that look interesting to test might be the expiry-based ones.